irodsbackend/static/irodsbackend/js/irodsbackend.js

```diff
@@ -254,11 +254,18 @@ $(document).ready(function() {
                         colSpan = '4';
                         icon = 'mdi:folder-open';
                         toolTip = 'Collection';
+
+                        var copyButton = $('<button>')
+                            .addClass('btn btn-secondary sodar-list-btn sodar-irods-copy-btn sodar-lz-copy-btn')
```

Comment:
It seems the bootstrap tooltips and the "copied!" notification from sodar-irods-copy-btn are not working. The `data-tooltip="tooltip" data-placement="top"` attributes seem to be missing.